main.cpp

```diff
@@ -79,42 +88,91 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+int openDoor(bool whoIS, int howMany)
+{
+    ignoreUnused(whoIS, howMany);
+    return{};
+}
+
 
 /*
  2)
  */
+int useComputer(bool available, int time, char application, char kindPC)
+{
+    ignoreUnused(available, time, application, kindPC);
+    return{};
+}
 
 /*
  3)
  */
-
+char letterRunner(bool importance, bool priority, char whosFor, char whosDelivering)
+{
+    ignoreUnused(importance, priority, whosFor, whosDelivering);
+    return{};
+}

```

Comment:
```
7) provide default values for an arbitrary number of parameters in the function parameter list.
```
I just need you to show me that you understand what 'default parameter value' means in terms of code.
- [ ] revise

Reply:
int useComputer(bool available = false, int time = 0, std::string application = "none", std::string kindPC = "Desktop")

'Default Parameter Value' is used to initialize the variable value.